src/components/tabController/TabBarItem.tsx

```diff
@@ -131,6 +131,9 @@ export default function TabBarItem({
   const sharedLabelStyle = useSharedValue(JSON.parse(JSON.stringify(labelStyle)));
   const sharedSelectedLabelStyle = useSharedValue(JSON.parse(JSON.stringify(selectedLabelStyle)));
 
+  const inactiveColor = useRef(_.cloneDeep(labelColor));
+  const activeColor = useRef(_.cloneDeep(!ignore ? selectedLabelColor : inactiveColor.current));
```

Comment:
I would give these consts a more specific name like 'animatedLabelColor' and 'animatedSelectedLabelColor' so it will be extra clear what this clone is for.

Reply:
I think the name is suffice, but I will add a note instead that explains the reason we keep it like this

Reply:
Added a note